docker/backend/Dockerfile

```
@@ -0,0 +1,123 @@
+# Set working directory
+WORKDIR /app
+
+# Copy dependency files from monorepo root
+COPY pyproject.toml uv.lock ./
+COPY README.md LICENSE ./
+
+# Copy backend source code from monorepo structure
+COPY src/backend/ ./src/backend/
+
+# Create virtual environment and install dependencies
+RUN python3.13 -m venv /app/.venv && \
```

Hadolint (2.14.0)

[error] Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead.

(SC2261)